client_test.go

```diff
@@ -8,10 +8,18 @@ import (
 )
 
 var client = NewClient("localhost:6379", "test_client", MakeStringPtr("SUPERSECRET"))
+var dummy = client.FlashAll()
```

Comment:
do we need this dummy? why not use a [blank identifier](https://golang.org/doc/effective_go.html#blank)?

Reply:
because this is the first time I'm writing in golang :)

Reply:
cool 👍